### PR TITLE
[IMP] base, point_of_sale, sale, website_sale: change tax label VAT -> Tax

### PR DESCRIPTION
**Pull request by hamo-odoo**
task-3653060

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by hamo-odoo** [comment]
I will be doing same for the other country in this PR

File: odoo/addons/base/models/res_country.py [hunk 1]
```diff
@@ -71,6 +71,7 @@ class Country(models.Model):
         ], string="Customer Name Position", default="before",
         help="Determines where the customer/company name should be placed, i.e. after or before the address.")
     vat_label = fields.Char(string='Vat Label', translate=True, prefetch=True, help="Use this field if you want to change vat label.")
+    tax_label = fields.Char(string='Tax Label', help="Use this field if you want to change tax label for Point of Sale")
```
Comment (ryv-odoo) [inline odoo/addons/base/models/res_country.py:74]: Why not adding this field in POS if it is the only usage ?

Comment (hamo-odoo) [reply odoo/addons/base/models/res_country.py:74]: I see a similar task(3516548) in progress may be I can rework with the help description but it is a reusable and helpful

Comment (hamo-odoo) [reply odoo/addons/base/models/res_country.py]: @ryv-odoo The above mentioned task is now moved to this task, I will be doing the same for E commerce and Sale Mail Template, I wonder if we are using VAT in any-other module too (cc- @jco-odoo)

Now this PR is not only about POS

File: addons/point_of_sale/views/report_saledetails.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@
                             <th class="text-start">Product Category</th>
                             <th class="text-start">Product</th>
                             <th class="text-end">Quantity</th>
-                            <th class="text-end">Total (VAT excluded)</th>
+                            <th class="text-end">Total (<t t-out="company.country_id.tax_label">VAT</t> excluded)</th>
```
[the remark this answers is not in the document]
Comment (hamo-odoo) [reply addons/point_of_sale/views/report_saledetails.xml:49]: As of what I know isn't here Python engine works rather than js?!

**Comment by Feyensv** [comment]
PR should stay in draft at least until it is validated (green 'Validation BE' stage).

**Comment by jco-odoo** [comment]
I wonder if we can not just change VAT -> Tax in these places as in a lot of cases you have the right naming already from the tax groups.  It is true however that not only India uses GST instead of VAT.  Another way might be to pass the GST countries in a list.  @dama-odoo @ryv-odoo @davidmonnom @hamo-odoo

**Comment by qle-odoo** [comment]
I am not convinced to add a field in base which is only used for one country.
other country does not use GST?

**Comment by hamo-odoo** [comment]
As of now we will be simply changing VAT -> Tax and as for the POS this fix would not work that great, because we will be needing tax bifurcation same as we get on the invoice as we had on the POS ticket back in 16.0 (it will be a new PR will try to target it in 17.0) and also now this PR won't add a new field we can re-target this PR to 17.0/saas-17.1 as we had POS REF in 17.1 (cc @kasp-odoo )

@qle-odoo @jco-odoo @dama-odoo @davidmonnom

**Comment by hamo-odoo** [comment]
@qle-odoo I have replaced VAT -> TAX as we discussed

**Comment by qle-odoo** [comment]
@robodoo r+

**Comment by Feyensv** [comment]
Where are the .pot updates if you're targeting 17.1 ?

@robodoo r-

**Comment by hamo-odoo** [comment]
@Feyensv @qle-odoo I have updated pot with latest file, though there seems to be extra diff please let me know, if it is not required

File: addons/sale/i18n/sale.pot [hunk 3]
```diff
@@ -4339,7 +4325,6 @@ msgstr ""
 #. module: sale
 #. odoo-python
 #: code:addons/sale/models/product_template.py:0
-#, python-format
```
Comment (Feyensv) [inline addons/sale/i18n/sale.pot:4342]: you can keep this change

File: addons/sale/i18n/sale.pot [hunk 4]
```diff
@@ -830,8 +830,7 @@ msgid ""
 "Quotation: the document will be sent to and accessible by customers at any time.\n"
 "e.g. this option can be useful to share Product description files.\n"
 "Confirmed order: the document will be sent to and accessible by customers.\n"
-"e.g. this option can be useful to share User Manual or digital content bought on ecommerce. \n"
-"Inside quote: The document will be included in the pdf of the quotation between the header pages and the quote table. "
+"e.g. this option can be useful to share User Manual or digital content bought on ecommerce. "
```
Comment (Feyensv) [inline addons/sale/i18n/sale.pot:833]: The exports were probably done on an instance with all modules installed.
(this one here comes from the module sale_pdf_quote_builder).

Remove the diff unrelated to your changes to be sure we don't impact anything else (only keep the unrelated python-format removal below, was not removed in a fw-port when it should have been).

**Review by Feyensv — COMMENTED** [review]
You can squash your two commits, there is no need to keep the two split as they are strictly related.

File: addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml [hunk 5]
```diff
@@ -64,9 +64,9 @@
             </t>
             <div t-if="props.data.tax_details.length > 0" class="pos-receipt-taxes">
                 <span />
-                <span>VAT%</span>
-                <span>VAT</span>
-                <span>ExVAT</span>
+                <span>Tax%</span>
```
Comment (hamo-odoo) [inline addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml:67]: Just wait need to check if this works well on POS
POS might have a spacing issue might need to replace it with CAPS

File: addons/sale/i18n/sale.pot [hunk 6]
```diff
@@ -3269,6 +3269,11 @@ msgstr ""
 msgid "Quotation #"
 msgstr ""
 
+#. module: sale
+#: model:ir.actions.report,name:sale.action_report_saleorder
+msgid "Quotation / Order"
+msgstr ""
+
```
Comment (Feyensv) [inline addons/sale/i18n/sale.pot:3276]: You can drop this part of the diff too ;)

File: addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml (hunk 5 above)
Comment (hamo-odoo) [reply addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml:67]: image
Seems like it is a CSS spacing issue, changing header to caps should work as of now :face_exhaling:

**Comment by hamo-odoo** [comment]
I hope, now we are good to go

**Comment by hamo-odoo** [comment]
@qle-odoo @Feyensv review+ ?

**Comment by qle-odoo** [comment]
@robodoo r+

**Pull request closed without merging**